### DISCUSSION TITLE
Bind Third Party login methods

### DISCUSSION THREAD

**Discussion by willin** [sergiodxa/remix-auth#259] [Q&A]
such as,
- user A, logged in with provider P1
- user A, bind with other provider P2

i planned to use a cookie to record user id and navigate to `auth.$provider.ts`.

however it just passed and do not redirect to provider P2's login page...

is there a way to let it come true.

**Comment by sergiodxa** [comment]
If you check what Kent does on the EpicStack, you can use `authenticator.authenticate` to authenticate the user but not set a successRedirect, then manually store the user in a different key than authenticator.sessionKey.

This will let you use Remix Auth to authenticate the user with the third-party provider but still consider the user unauthenticated, which will let you keep asking the user to authenticate with a third-party provider.
